pkg/console/install_panels.go

```diff
@@ -1501,11 +1554,16 @@ func addInstallPanel(c *Console) error {
 
 				if needToGetVIPFromDHCP(c.config.VipMode, c.config.Vip, c.config.VipHwAddr) {
 					printToPanel(c.Gui, "Configuring network...", installPanel)
-					if _, err := applyNetworks(c.config.Networks, c.config.Hostname); err != nil {
+					if _, err := applyNetworks(c.config.ManagementInterface, c.config.Hostname); err != nil {
 						printToPanel(c.Gui, fmt.Sprintf("can't apply networks: %s", err), installPanel)
 						return
 					}
-					vip, err := getVipThroughDHCP(config.MgmtInterfaceName)
+					mgmtName := config.MgmtInterfaceName
+					vlanID := c.config.ManagementInterface.VlanID
+					if vlanID >= 2 && vlanID <= 4094 {
+						mgmtName = fmt.Sprintf("%s.%d", mgmtName, vlanID)
+					}
+					vip, err := getVipThroughDHCP(mgmtName)
```

Comment:
This is re-used in another place, might be good to implement it as a method in Network.

Reply:
fixed